website/server/middlewares/cron.js

```diff
@@ -6,9 +6,10 @@ import Bluebird from 'bluebird';
 import { model as Group } from '../models/group';
 import { model as User } from '../models/user';
 import { recoverCron, cron } from '../libs/cron';
-import { v4 as uuid } from 'uuid';
 
 const daysSince = common.daysSince;
+// Wait this length of time in ms before attempting another cron
+const cronTimeout = new Date(3600000).getTime();
```

Comment:
we try to name constants with all upper case letters, so this would be CRON_TIMEOUT. Also for the length of time maybe a simple, 1000 * 60 * 60 is easier to read (with a comment saying that it's 1h)